src/styles/blocks/header.scss

```diff
@@ -0,0 +1,99 @@
+* {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+:root {
+  --header-height: 60px;
+}
+
+.roboto-medium {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  margin: 0 auto;
+  padding: 0 50px;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  text-transform: uppercase;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  color: black;
+  transition: all ease-in-out 0.15s;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__item:hover {
+  color: #00acdc;
+}
+
+.nav__link {
+  font-weight: 500;
+  font-size: 12px;
+  height: var(--header-height);
+  line-height: var(--header-height);
+  font-family: Roboto, sans-serif;
+  text-decoration: none;
+  display: inline-block;
+  color: #060b35;
```

Comment:
The `.nav__link` class duplicates the font properties defined in both `html` and `.roboto-medium`. To keep your code DRY, you can extend `.nav__link` from `.roboto-medium` or remove the redundant font declarations if the `html` font style is applied project-wide.